resolwe/flow/executors/run.py

```diff
@@ -151,6 +151,8 @@ def _run(self, data_id, script, verbosity=1):
         )
 
         # Run process and handle intermediate results
+        logger.info("Running program for Data with id {}".format(data_id))
+        logger.debug("The program for Data with id {} is: \n{}".format(data_id, script))
         self.run_script(script)
         spawn_processes = []
         output = {}
```

Comment:
Since you are improving logging in the executor, also fix the line `logger.error("Out of memory: %s", ex)` and use `logger.exception`.

Reply:
Could you also replace `logger.error("Error sending command to manager:\n\n%s", traceback.format_exc())` in `manager_commands.py`?

Reply:
I've replaced `%s` with `.format`, but that's all that can be done as the protocol doesn't support sending tracebacks from executor to manager.